implementation/exporters/src/main/java/io/smallrye/opentelemetry/implementation/exporters/sender/VertxGrpcSender.java

```diff
@@ -58,17 +58,21 @@ public final class VertxGrpcSender<T extends Marshaler> implements GrpcSender<T>
     private final SocketAddress server;
     private final boolean compressionEnabled;
     private final Map<String, String> headers;
+    private final String signalType;
     private final String grpcEndpointPath;
 
     private final GrpcClient client;
 
     public VertxGrpcSender(
+            String signalType,
```

Comment:
Why do you need a signal type? The marshaller will handle that.

Reply:
It's used in the property lookup to get signal-specific configurations. See here, for example: https://github.com/smallrye/smallrye-opentelemetry/pull/402/files#diff-a90e43f59d93847ebb3a932a020d18b407e60ab8325d6839be09ec3fcc30be19R73-R79

Reply:
@brunobat *bump* I need to make a release for WildFly, and it would be nice to include this change if you're comfortable with it.